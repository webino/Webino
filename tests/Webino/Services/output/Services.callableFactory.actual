/* Closure defined in file /home/linux/Webino/www/Webino/tests/Webino/Services/Services.callableFactory.phpt on line 27 */
