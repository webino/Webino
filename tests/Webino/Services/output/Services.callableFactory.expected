TestService::__set_state(/* #026e */ array())
